src/composables/gmnh-service.ts

```diff
@@ -0,0 +1,49 @@
+import { DEFAULTS } from "@/globals";
+import {PNFT} from "@/common/helpers/types"
+
+export async function getOpenQuestionsFromGMNH(): Promise<PNFT[]> {
+    const pnfts = await retrieveOpenQuestions();
+    return pnfts;
+}
+
+async function retrieveOpenQuestions(): Promise<PNFT[]> {
+    return fetch('http://gmnh-service.herokuapp.com/openquestions')
+            // the JSON body is taken from the response
+            .then(res => res.json())
+            .then(res => {
+                return res as PNFT[]
+            })
+}
+
+export async function getAnsweredQuestionsFromGMNH(): Promise<PNFT[]> {
+    const pnfts = await retrieveAnsweredQuestions();
+    return pnfts;
+}
+
+async function retrieveAnsweredQuestions(): Promise<PNFT[]> {
+    return fetch('http://gmnh-service.herokuapp.com/answeredquestions')
+            // the JSON body is taken from the response
+            .then(res => res.json())
+            .then(res => {
+                return res as PNFT[]
+            })
+}
+
+export async function getMyQuestionsFromGMNH(userWalletAddr: string): Promise<PNFT[]> {
+    const pnfts = await retrievMyQuestions(userWalletAddr);
+    return pnfts;
+}
+
+async function retrievMyQuestions(userWalletAddr: string): Promise<PNFT[]> {
+    let myQuestionsUrl = 'http://gmnh-service.herokuapp.com/myquestions' + '/' + userWalletAddr;
```

Comment:
can de-duplicate having written the app url 3 times if we saved in 1 var at the top of this file

Reply:
This is a good point.  We'll eventually need to make this an env var anyways.  let me adjust!